Package.swift

```diff
@@ -9,7 +9,7 @@ let package = Package(
         .library(name: "Shout", targets: ["Shout"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/IBM-Swift/BlueSocket", from: "0.12.68"),
+        .package(url: "https://github.com/IBM-Swift/BlueSocket", from: "1.0.0"),
```

Comment:
Why from 1.0.0? I'd set it to be from 1.0.46, the most recent version available.

Reply:
+1